src/annotator/guest.coffee

```diff
@@ -37,7 +38,7 @@ module.exports = class Guest extends Delegator
     TextSelection: {}
 
   # Anchoring module
-  anchoring: require('./anchoring/html')
+  anchoring: htmlAnchoring
```

Comment:
The new approach to mocking only currently supports dependencies declared at the top level of the module, so this hoists the `require` to the top of the file. This is needed anyway if we later convert this module to JS with `import` and `export` statements, since `import` can only appear at the top level as well.